model.py

```diff
@@ -1,19 +1,56 @@
+from dao import Dao
+
 class Model:
+    data_access = Dao()
 
-    def __init__(self, name, function):
+    def __init__(self, name, function, _id):
         self.name = name
         self.function = function
+        self._id = _id
+    @classmethod
+    def update(cls, name, function):
+        thing = cls.data_access.delete(name)
+        if not thing:
+            return None
+        updated_thing = {'name':name, 'function':function}
+        cls.data_access.add(updated_thing)
+        return cls.from_dict(updated_thing)
+        
+    @classmethod
+    def save(cls, name, function):
```

Comment:
save with a payload instead of individual parameters
```py
def save(cls, id, payload)
```

Reply:
done